src/app/Types.elm

```diff
@@ -51,3 +53,4 @@ type Msg
     | SetCup CupOfDice
     | NextServerState ServerState
     | RoundFinished RoundResult
+    | TableMsg Table.Msg
```

Comment:
Ah ok. Sorry, I was just looking at the app in the current state (because it's the code I'm submitting this PR against). The `Msg` type it uses is the one in the `Types` module called `Msg`, so that's where I put it. 👍